src/modules/RA/Viewpoint/views/Create.js

```diff
@@ -9,7 +9,7 @@ export const ViewpointCreate = props => (
     {...props}
     actions={<DefaultActions />}
   >
-    <ViewpointFields />
+    <ViewpointFields create />
```

Comment:
I don't see the point of this prop...

Reply:
Is create equivalent to !edit. In this case I agree.

Reply:
Done